<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friend</class>
 <widget class="QWidget" name="Friend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Linpop</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:vertical {
	width: 10px;
	border: 0px;
}

QScrollBar:vertical:hover {
	background: rgb(221,225,228);
}

QScrollBar::handle:vertical {
	background: rgb(198,201,203);
	border-radius:5px;
}

QScrollBar::add-line:vertical {
	border: 0px;
	height: 0px;
}

QScrollBar::sub-line:vertical {
	border:0px;
	height: 0px;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	border: 0px;
}

QScrollBar::add-page:vertical{
	border-top-left-radius:5px;
	border-top-right-radius:5px;
}

QScrollBar::sub-page:vertical {
	border-bottom-left-radius:5px;
	border-bottom-right-radius:5px;
}

QScrollBar::add-page:vertical:hover, QScrollBar::sub-page:vertical:hover {
	background: rgb(221,225,228);
}

QScrollBar:horizontal {
	border: 0px;
	height: 10px;
}

QScrollBar::handle:horizontal {
	background: rgba(69,124,216,150);
}

QScrollBar::add-line:horizontal {
	border: 0px;
	width: 0px;
}

QScrollBar::sub-line:horizontal {
	border:0px;
	width: 0px;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
	border: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: rgba(0,0,0,0);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#horizontalWidget {
	background-image: url(:/friend/bg.png) 0 72 0 36;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>11</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QPushButton" name="iconButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#iconButton {
    border-image: url(://icon/icon1.png);
	background-color: rgba(255, 255, 255, 0);
	border-radius: 3px;
}

QPushButton#iconButton:focus {
	border:3px solid rgb(213, 213, 213);
	border-radius: 12px;
}

QPushButton#iconButton:hover {
	border:3px solid rgba(0, 0, 0, 100);
	border-radius: 12px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="horizontalWidget_2" native="true">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="nameEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	font-size: 10pt;
	background-color: rgba(255, 255, 255, 0);
	border:none;
	border-radius:2px;
}

QLineEdit:focus{
	background-color: rgba(255, 255, 255);
	border:1px solid rgb(213, 213, 213);
}

QLineEdit:hover{
	 border:1px solid rgba(0, 0, 0, 100);
}</string>
              </property>
              <property name="text">
               <string>张三</string>
              </property>
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="idLabel">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>10001</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#horizontalWidget_3 {
	background-color: rgb(215, 215, 215);
}

#horizontalWidget_3 QLabel {
	background-color: rgba(0, 0, 0, 0);
}

QLabel#label_3 {
	border: 2px;
	border-right-color: rgba(0, 0, 0, 0);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>    好友</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-right:1px solid rgba(0, 0, 0, 60);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>135</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>135</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>    IP地址</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="friendWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
